fixtures/enum-types/tests/bindings/test_enum_types.py

```diff
@@ -0,0 +1,14 @@
+import unittest
+from enum_types import *
+
+assert(Animal.DOG.value == 0)
+assert(Animal.CAT.value == 1)
+
+assert(AnimalInt.DOG.value == 3)
```

Comment:
It won't work for kotlin or swift here since there is no `Repr` (which ty for suggestion, I'll rename this enum so it's more obvious) and those languages don't default to int for the underylings. One option is to default everything to unsigned if not defined but that could have breaking implications for people using the enum proc-macro right now (if not using unsigned literals).